captura/test_views.py

```diff
@@ -0,0 +1,67 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.core.urlresolvers import reverse
+from splinter import Browser
+
+
+class TestViewsCaptura(StaticLiveServerTestCase):
+    """Integration test suite for testing the views in the app: captura.
+
+    Test the urls for 'captura' which make up the captura dashboard.
+
+    Attributes
+    ----------
+    browser : Browser
+        Driver to navigate through websites and to run integration tests.
+    """
+
+    def setUp(self):
+        """Initialize the browser and create a user, before running the tests.
+
+        """
+        self.browser = Browser('chrome')
```

Comment:
you are not using this browser in any test. thus, you should not declare it. The same for every test you did.